vertical-pod-autoscaler/deploy/vpa-rbac.yaml

```diff
@@ -82,6 +82,13 @@ rules:
       - create
       - patch
       - delete
+  - apiGroups:
```

Comment:
I don't think this belongs into the `system:vpa-checkpoint-actor` `ClusterRole` – this role only deals with access to `VPACheckpoints`. The role `system:vpa-actor` seems like a better semantical fit.
The only issue is that this role is also given to the `vpa-updater`, which doesn't need access to the `/status` subresource.

So probably we should introduce a new ClusterRole with name `system:vpa-status-actor` here, to keep things minimal.

Reply:
done